### PR TITLE
split components

### PR DESCRIPTION
**Pull request by tokinonagare**

### CODE REVIEW

**Review by joebanezair — APPROVED** [review]
Approved PR

File: src/view/UnBundEmailAddress.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import React, { useState } from "react";
+import { View } from "react-native";
+import { TouchableOpacity, Text, Alert, TextInput, Modal } from 'react-native';
+import styles, {textStyle} from "../style/UnBundEmailAddressStyle";
+
+const UnBundEmailAdress =()=>{
```
Comment (tokinonagare) [inline src/view/UnBundEmailAddress.js:6]: Spelled wrong

Comment (tokinonagare) [reply src/view/UnBundEmailAddress.js:6]: The address of the email address needs to be passed through the parent component (no need to change it by clicking)

File: src/view/UnBundEmailButton.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import React, { useState } from "react";
+import { TouchableOpacity, Text, Alert } from 'react-native';
+import styles from "../style/UnBundEmailButtonStyle";
+
+const UnBundButtonEmail = () => {
+  const [isButtonPressed, setIsButtonPressed] = useState(false);
+
+  const handleButtonPress = () => {
```
Comment (tokinonagare) [inline src/view/UnBundEmailButton.js:8]: This click event needs to expose an interface to the parent component (remember that any interaction should not be placed in the view)

**Pull request merged**